src/App.tsx

```diff
@@ -1,173 +1,49 @@
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person" className="has-background-warning">
-                  <td>
-                    <a href="#/people/jan-frans-van-brussel-1761">
-                      Jan Frans van Brussel
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1761</td>
-                  <td>1833</td>
-                  <td>-</td>
-
-                  <td>
-                    <a href="#/people/jacobus-bernardus-van-brussel-1736">
-                      Jacobus Bernardus van Brussel
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
-          </div>
-        </div>
-      </div>
-    </main>
+    <Routes>
+      <Route path="/" element={<Layout />}>
+        <Route
+          index
+          element={<h1 className="title">Home Page</h1>}
+        />
+        <Route
+          path="*"
+          element={<h1 className="title">Page not found</h1>}
+        />
+      </Route>
+
+      <Route
+        path="home"
+        element={<Navigate replace to="/" />}
+      >
+        <Route
+          path="*"
+          element={<h1 className="title">Page not found</h1>}
+        />
+      </Route>
+
+      <Route path="people" element={<Layout />}>
+        <Route path=":slug?" element={<PeoplePage />} />
+        <Route
+          path="*"
+          element={<h1 className="title">Page not found</h1>}
+        />
+      </Route>
+
+      <Route
+          path="*"
+          element={<h1 className="title">Page not found</h1>}
+        />
+    </Routes>
```

Comment:
- It's good practice to move all your routes to the Root component
- Keep the DRY principle (you repeat the route for NotFoundPage several times)